modules/serverless-connect-func-source-odc.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-eventing.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-connect-func-source-odc_{context}"]
+= Connect an event source to a function using the Developer perspective
+
+Functions are deployed as Knative services on an {product-title} cluster. When you create an event source by using the {product-title} web console, you can specify a deployed function that events are sent to from that source.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
+* You have logged in to the web console and are in the *Developer* perspective.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created and deployed a function.
+
+.Procedure
+
+. Create an event source of any type, by navigating to *+Add* -> *Event Source* and then selecting the event source type that you want to create.
+
+. In the *Sink* section of the *Create Event Source* form view, select your function in the *Resource* list.
+
+. Click *Create*.
+
+.Verification
+
+You can verify that the event source was created and is connected to the function by viewing the *Topology* page.
+
+. In the *Developer* perspective, navigate to *Topology*.
+
+. View the event source and click on the connected function to see the function details in the side panel.
```

Comment:
```suggestion
. View the event source and click on the connected function to see the function details in the right panel.
```

Reply:
Not sure of the UI, so it might not be the 'right panel', but in the ISG, it gives examples of using 'panel', 'left panel', and 'right panel'. See if this applies here...
https://www.ibm.com/docs/en/ibm-style?topic=grammar-prepositions